### DISCUSSION TITLE
Apply filter action to updated articles too

### DISCUSSION THREAD

**Discussion by hkcomori** [FreshRSS/FreshRSS#6330] [Ideas]
When 'Mark updated articles as unread' triggered, the filter action is not applied.
In my usecase, I set it for articles not wanted to see, so I expect only articles that were not filtered to be unread.

How about your use cases?

**Comment by Alkarex** [comment]
Indeed, this should be improved.

The difficulty with applying actions on updated articles is to make sure that we do not break manual user actions. In particular, I do not think we should re-apply the automatic labelling on updated articles (that would require a much more complicated logic to find out whether the article had been tagged and untagged before).

But your specific usecase should be fine to implement.

**Reply by Alkarex** [reply]
@hkcomori Would you be able to test https://github.com/FreshRSS/FreshRSS/pull/6334 ?

**Reply by hkcomori** [reply]
Thanks!
Can I close this discussion because we can track it in #6331?
